### PR TITLE
[JAVA API] Java samples build with gradle, Kotlin sample

### PR DESCRIPTION
**Pull request by likholat**

### CODE REVIEW

**Comment by likholat** [comment]
@dkurt

File: modules/java_api/org/intel/openvino/Core.java [hunk 1]
```diff
@@ -41,6 +41,7 @@ private static String getLibraryName(String name, String linux_ver) {
 
     // Use this method to initialize native libraries and other files like
     // plugins.xml and *.mvcmd in case of the JAR package os OpenVINO.
+    // Doesn't work for Android!
```
Comment (dkurt) [inline modules/java_api/org/intel/openvino/Core.java:44]: Maybe we can just detect Android system and do loadLibrary inside?

Comment (likholat) [reply modules/java_api/org/intel/openvino/Core.java:44]: Maybe in the next PR

**Comment by alvoron** [comment]
@likholat could you please resolve conflict?

**Comment by alvoron** [comment]
@dkurt are you OK to merge this PR?

File: modules/java_api/org/intel/openvino/Core.java [hunk 2]
```diff
@@ -47,10 +47,9 @@ private static String getLibraryName(String name, String linux_ver) {
         return name;
     }
 
-    /**
-     * Use this method to initialize native libraries. Other files like plugins.xml and *.mvcmd will
-     * be also copied to temporal location which makes them visible in runtime.
-     */
+    // Use this method to initialize native libraries and other files like
+    // plugins.xml and *.mvcmd in case of the JAR package os OpenVINO.
+    // To Do: fix for Android!
```
Comment (dkurt) [inline modules/java_api/org/intel/openvino/Core.java:52]: Why docstring has been changed from `/**/` to `//`?

**Comment by dkurt** [comment]
@likholat, @alvoron, sorry for delay. PR looks good to me except one [comment](https://github.com/openvinotoolkit/openvino_contrib/pull/432#discussion_r985450781). Please take a look.

File: modules/java_api/org/intel/openvino/Core.java (hunk 2 above)
Comment (likholat) [reply modules/java_api/org/intel/openvino/Core.java]: Thanks, fixed

**Review by dkurt — APPROVED** [review]

**Review by alvoron — APPROVED** [review]

**Pull request merged**